influxdb3_write/src/write_buffer/mod.rs

```diff
@@ -507,21 +511,21 @@ pub(crate) fn validate_or_insert_schema_and_partitions(
 // &mut Cow is used to avoid a copy, so allow it
 #[allow(clippy::ptr_arg)]
 fn validate_and_convert_parsed_line<'a>(
-    line: ParsedLine<'_>,
+    mut line: ParsedLine<'_>,
     raw_line: &'a str,
     segment_table_batches: &mut HashMap<Time, TableBatchMap<'a>>,
     schema: &mut Cow<'_, DatabaseSchema>,
     ingest_time: Time,
     segment_duration: SegmentDuration,
     precision: Precision,
 ) -> Result<()> {
-    let table_name = line.series.measurement.as_str();
+    let table_name = line.series.measurement.to_string();
```

Comment:
I don't think this needs to be a string. I don't see any changes you've made that would require that, unless something was causing the borrow checker to complain that I'm missing.

Reply:
I changed this to satisfy the borrower checker originally, but I will take a second look if this is needed.